### ISSUE TITLE
Screens refuse to show anything other than 'NO DATA'

### ISSUE THREAD

**Issue by StardustAlpha** [SonarSonic/Practical-Logistics-2#38]
[Minecraft 1.10.2]
No matter what I do, nothing happens. It just says NO DATA.
![Screenshot](https://user-images.githubusercontent.com/28673956/29986800-38c93fce-8f22-11e7-9757-e17870135943.png)

**Comment by chezpaul** [comment]
Same problem here, using 2.0.2 version with latest forge on 1.10.2
Are you still working on this mod? 
It looks really cool but doesn't seem to work.
I was using wireless emitters and receiver with trying to read a vibrant ender energy storage.
Receiver does see the storage, I can select it but it doesn't give me any other options.
Screen stays blank, NO DATA.

On emitter I have it coming from a energy reader.

**Comment by chezpaul** [comment]
BTW, this also happens when plugged direct without the wireless stuff.

**Comment by Cmoor** [comment]
I can't see properly how you have the setup but are you using an info/inventory reader?
It doesn't seem like it. From the chest it seems you only have a node going into a screen. You need the reader to actually tell the screen what to display
Maybe another view of your setup?

**Comment by chezpaul** [comment]
Yeah his looks like it's wrong but mine was set right. Node, cable, energy reader, cable wireless emitter
